### ISSUE TITLE
Linking from the documentation to the source code

### ISSUE THREAD

**Issue by ghuntley** [reactiveui/documentation#23]
Not sure yet what the best way to handle this will be.

# Conversation

Michael
> How do you feel about linking from docs to source code on GH? For example, from the `Interaction` docs to the class?

Geoffrey
> There's no path from docs to code. There is a path from VS symbol direct to github source code for that symbol at that specific commit from v7 packages. GitLink. That's the preferred and suggested way. New up the symbol and view source within VS. 
> See https://github.com/GitTools/GitLink/blob/develop/README.md
> As for linking into class. Getting people to read the source is definitely something we want people to be doing at this stage of the project (minimal docs + samples). How to encourage and make this happen. Not sure. Like the source code is right there on GitHub....

Michael
> We could add a footer: ReacticeUI source code (linked to https://github.com/ReactiveUI/ReactiveUI)

**Comment by moswald** [comment]
Drive-by commenting!

I'm all for a link back to the comments. It would be best if we linked directly to the class, and I think RxUI is stable enough that for the most part, they won't become stale. For example, `ReactiveObject` is likely to always be in the same file in the same folder, no?

Another option would be to use an automatically generated list of APIs, something like what FAKE does. I think [FAKE's documentation](http://fsharp.github.io/FAKE/) leaves a lot to be desired, but I can always answer my own questions by browsing the [API list](http://fsharp.github.io/FAKE/apidocs/index.html).

**Comment by ghuntley** [comment]
What about supplementing the documentation with a separate resource just for discovering/exploring the API surface? http://reactiveui.net/api/*

Check out http://reactiveui.net/api/index.html?%2Fapi%2Fpages%2FReactiveUI.MessageBus.html

**Comment by kentcb** [comment]
Yeah, I like the idea of links directly from the API docs to the relevant source, but only if we can do it without risk of stale links etc. In the absence of any tooling that makes this viable, I'd say two links - one to source and one to API docs - is sufficient.

PS. unless SharpDox has come a long way since I last used it (around a year or two ago), it may not be a contender. There is a more "official"

**Comment by mteper** [comment]
I like it.
